timer/src/org/jeffpiazza/derby/devices/MiscJunkDevice.java

```diff
@@ -78,7 +78,10 @@ public boolean probe() throws SerialPortException {
                                    /* dtr */ false)) {
       return false;
     }
-
+    // We just reset the timer, give it 2s to startup
+    try {
+       Thread.sleep(2000); // ms.
```

Comment:
Does the PDT actually reset here?  All the probe is actually doing is changing the communication settings on the host-side port.  I'm not sure how the PDT can even be aware this is happening.

Reply:
Yes, the PDT does reset.  This is shown in the derby-timer logs indicating receipt of "P" for power-on indicating the PDT code as restarted as well as the PDT status LED which turns off for a brief time when scanning starts.

The PDT is powered via the same USB connection used to communicate with derby-timer.